### PR TITLE
ESLint migration 🚗 (WIP)

### PR DESCRIPTION
**Pull request by amilajack**
**Please don't merge this yet, as this is a work in progress**

### CODE REVIEW

**Comment by amilajack** [comment]
Why was this closed?

**Comment by amilajack** [comment]
@mathiasrw You deleted the dev-master branch and deleted all the work I was doing on this. I've done about 2000+ changes. Now I have to start over again. Please ask before deleting. Not asking others before making decisions defeats the purpose of having a team in the first place.

**Comment by mathiasrw** [comment]
>  Not asking others before making decisions defeats the purpose of having a team in the first place.

I agree. My bad. Wont happen again. 

Is there any connection between the deleted `dev-master` you mention and the still existing `eslint-migration` that this PR was for?

**Comment by amilajack** [comment]
`eslint-migration` is a branch that I pushed from my local machine. It is an equivalent of the previous `origin/eslint-migration`.

**Comment by mathiasrw** [comment]
Wait - `eslint-migration` was also gone???

It all starts go make more sense now why you are upset - until now I have only had `dev-master` in mind but now that you tell me is that `eslint-migration` was also gone I have investigated a bit. 

**tl;dr**
- I know for sure i did not touch `eslint-migration` 
- I was thinking that deleting dev-master would be reasonable
- Checking my bash history shows i did not delete any branch at all.

Conslusion: someone else cleaned up the branches. 

**Long version:**

My bash history is posted here - but nowhere is there a `git bash -dr <branch-name>`

- 283-301: Yesterday - setting up node on (setting up partners computer - my own was in the car)
- 302-427: Yesterday - trying to merge MaXwellFalsteins commits into this tree
- 428-438: Yesterday - end up making a dedicated branch to MaXwellFalstein
- 439-444: Today - looking into what data I got in my repo

```
  283  node
  284  brew update
  285  /usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
  286  node -v 
  287  npm 
  288  npm -v
  289  npm update npm
  290  npm update npm -g
  291  sudo npm update npm -g
  292  brew install  node
  293  mkdir git
  294  cd git
  295  ls
  296  git clone https://github.com/parallel-js/parallel.js
  297  cd parallel.js/
  298  ls
  299  branch
  300  git log
  301  brew install git-flow 
  302  git flow init
  303  git branch
  304  git branch -d develop
  305  git checkout master
  306  git branch -d develop
  307  git branch MaXwellFalstein 0747cd5
  308  git branch
  309  git branch develop 0747cd5
  310  git flow feature start MaXwellFalstein
  311  git branc
  312  git branch
  313  git branch -d MaXwellFalstein
  314  vim .git/config
  315  git checkout -b MaXwellFalstein origin2/master
  316  git fetch origin2/master
  317  git fetch MaXwellFalstein
  318  git fetch https://github.com/MaXwellFalstein/Parallel2.js
  319  git status
  320  git checkout -b MaXwellFalstein origin2/master
  321  git branch
  322  git checkout -b MaXwellFalstein origin2/mastergit remote -v
  323  git git remote -v
  324  git remote -v
  325  git fetch origin2
  326  git checkout -b MaXwellFalstein origin2/master
  327  git status
  328  git log
  329  git branch
  330  git checkout feature/MaXwellFalsterin
  331  git checkout feature/MaXwellFalstein
  332  git rebase maxWellFalstein
  333  open package.json 
  334  open README.md 
  335  git status
  336  git rebase --continue
  337  git add README.md 
  338  git add package.json 
  339  git rebase --continue
  340  open .gitignore
  341  git rebase --continue
  342  git add .gitignore
  343  git rebase --continue
  344  git stash
  345  git status
  346  open .gitignore
  347  git status
  348  git add .gitignore
  349  git rebase --continue
  350  open package.json 
  351  git add package.json 
  352  git rebase --continue
  353  open README.md 
  354  git add README.md 
  355  git rebase --continue
  356  open package.json 
  357  git add package.json 
  358  git rebase --continue
  359  open package.json 
  360  git add package.json 
  361  git rebase --continue
  362  git status
  363  git open lib/Worker.js~HEAD
  364  open lib/Worker.js~HEAD
  365  open lib/Worker.js
  366  git rebase --continue
  367  open README.md 
  368  git add README.md 
  369  git rebase --continue
  370  git add README.md 
  371  open README.md 
  372  git add README.md 
  373  git rebase --continue
  374  open package.json 
  375  git add README.md 
  376  git add package.json 
  377  git rebase --continue
  378  git rebase --aboard
  379  git rebase --aboart
  380  git rebase --abort
  381  git status
  382  git stash
  383  git branch
  384  git >>>>>>> Renames the project to thread.js.
  385  git checkout MaXwellFalstein
  386  git log
  387  git branch
  388  git checkout feature/MaXwellFalstein
  389  git cherry-pick 
  390  git cherry-pick f1c97777cbe4faf031f90611809f806c51be477b
  391  git cherry-pick de11b5d539b89287ee84f03cd538f7d12d63b63b
  392  git status
  393  git stash
  394  git cherry-pick de11b5d539b89287ee84f03cd538f7d12d63b63b
  395  git status
  396  git stasu
  397  git stash
  398  git reset --soft HEAD
  399  git reset --soft HEAD~1
  400  git merge --abort
  401  git cherry-pick  --abort
  402  git reset --soft HEAD~1
  403  git cherry-pick de11b5d539b89287ee84f03cd538f7d12d63b63b
  404  git statys
  405  git status
  406  git stash
  407  git cherry-pick de11b5d539b89287ee84f03cd538f7d12d63b63b
  408  git status
  409  git stash
  410  git status
  411  git stash
  412  git branch
  413  git checkout MaXwellFalstein
  414  git flow
  415  git fow feature
  416  git flow feature
  417  git branch -d feature/MaXwellFalstein
  418  git checkout develop
  419  git branch -d feature/MaXwellFalstein
  420  open README.md 
  421  git status
  422  git add README.md 
  423  git merge --continue
  424  git cherry-pick --continue
  425  git checkout develop
  426  git branch -d feature/MaXwellFalstein
  427  git branch -D feature/MaXwellFalstein
  428  git branch MaXwellFalstein
  429  git checkout MaXwellFalstein
  430  git flow start MaXwellFalstein
  431  git flow feature start MaXwellFalstein
  432  git checkout MaXwellFalstein
  433  git branch -D feature/MaXwellFalstein
  434  git branch -m MaXwellFalstein feature/MaXwellFalstein
  435  git push feature/MaXwellFalstein origin
  436  git push origin
  437  git config --global push.default simple
  438  git push origin
  439  cd git
  440  cd parallel.js/
  441  git branch
  442  cgit branch master
  443  git branch master
  444  git checkout  master
```

**Comment by amilajack** [comment]
I'm not angry/mad and I'm not trying to blame anyone. I don't really care who deleted the branch. Just saying that we need some more communication between maintainers
